css/main.css

```diff
@@ -0,0 +1,303 @@
+body {
+  font-family: Raleway;
+  margin: 0;
+  background-color: rgb(241, 241, 241);
+}
+
+a, a:visited, a:hover {
+  color: black;
+  text-decoration: none;
+}
+
+/* h1 not bold standard */
+h1, h3 {
+  font-weight: normal;
+}
+
+h2 {
+  text-transform: uppercase;
+}
+
+h2 span {
+  font-weight: 100;
+}
+
+h3 {
+  font-size: 150%;
+}
+
+.up-button {
+  position: fixed;
+  bottom: 20px;
+  right: 20px;
+  width: 30px;
+  height: 30px;
+  background-color: white;
+  border: 1px solid black;
+  text-align: center;
+  line-height: 30px;
+}
+
+/* ----------------------------- */
+/* header menu */
+/* ----------------------------- */
+.wrapper-header {
+  border-bottom: 1px solid rgba(0,0,0,0.8);
+  box-shadow: 0px 4px 16px 0px rgba(0,0,0,0.35); 
+  padding-top: 10px;
+  background-color: white; 
+}
+
+header {
+  width: 70%;
```

Comment:
I think this this width property gives you unintended results.   I would leave the width of the header and the nav alone, allowing them to take 100% width by default.  Actually I'm not sure you need to nest your nav inside a header.  Once your navigation bar is taking 100% width with  `display: flex` then you can use `justify-content: space-around` to better effect (check your page with smaller screen size with the dedicated button in chrome dev tools (top left 😉)